client/state/initial-state.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore } from 'state';
```

Comment:
This could be consolidated to `import { createReduxStore, reducer } from 'state';`